yaroslav.nazarenko/first/task1/task1.rb

```diff
@@ -0,0 +1,2 @@
+x = Array.new(3){Array.new(3){rand(-10..10)}}
+x.flatten.each {|x| puts x if x>0}
```

Comment:
You also need to add spaces here
```suggestion
x.flatten.each { |x| puts x if x > 0 }
```